_src/_includes/sidenote.html

```diff
@@ -0,0 +1 @@
+<label for="{{ include.reference }}" class="margin-toggle sidenote-number"></label><input type="checkbox" id="{{ include.reference }}" class="margin-toggle"/><span class="sidenote">{{ include.content }}</span>
```

Comment:
This is a bit tough to read all on one line. Does it impact the resulting Markdown output if the code is formatted correctly?